Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -221,6 +225,8 @@ screen initialState =
             case initialState.props.currentStage of
               SearchLocationModel -> case initialState.props.isSearchLocation of
                 LocateOnMap -> do
```

Comment:
we can move the functionality inside LocateOnMap inside a function as it is being used at multiple places